groupthink/commands.py

```diff
@@ -2,107 +2,129 @@
+storage = '{}/.groupthink'.format(home)
+
+@arg('group', help='The group to install')
+@arg('-a','--alias', help='Set a different name for this command')
+@arg('-d','--dest')
+@arg('-s', '--storage')
+def install(group, alias=None, dest=dest, storage=storage):
+    # install group scripts repository in ~/.groupthink
     # and add groupthink script to /usr/local/bin
-    alias = alias if alias else org
-    check_install(org, check_already=False, repo_dir=repo_dir)
-    script_exists = os.path.exists('{0}/{1}'.format(script_dir, org))
+    alias = alias if alias else group
+    check_install(alias, check_already=False, storage=storage)
+    script_exists = os.path.exists('{0}/{1}'.format(dest, alias))
     if script_exists:
         message = []
-        message.append("A script is already installed with the name {0}.".format(org))
-        message.append("You can try to install {0}-cli under a different name, with:\n".format(org))
-        message.append("  groupthink --install name --alias {0}".format(org))
+        message.append('A script is already installed with the name {0}.'.format(alias))
+        message.append('You can try to install {0}-cli under a different name, with:\n'.format(group))
+        message.append('  groupthink install {0} --alias ALIAS'.format(group))
         return '\n'.join(message)
     try:
-        execute_cmd(['mkdir', '-p', repo_dir])
-        repo_url = 'https://github.com/{0}/{0}-cli.git'.format(alias)
-        repo_dest = '{}/{}-cli'.format(repo_dir, org)
-        (msg, err) = execute_cmd(['git', 'clone', repo_url, repo_dest])
-        if err.find('fatal: ') > -1:
-            sys.exit(1)
+        execute_cmd(['mkdir', '-p', storage])
+        repo_url = 'https://github.com/{0}/{0}-cli.git'.format(group)
+        repo_dest = '{}/{}-cli'.format(storage, group)
+        if requests.get(repo_url[:-4]).status_code < 400:
+            (msg, err) = execute_cmd(['git', 'clone', repo_url, repo_dest])
+            if err.find('fatal: ') > -1:
+                return '{0}-cli is already installed. Remove it before creating an aliased command.'.format(group)
+                sys.exit(1)
+        else:
+            return 'Could not get {0}-cli command. Please make sure https://github.com/{0}/{0}-cli exists.'.format(group)
         groupthink_script = os.path.dirname(os.path.realpath(__file__)) + '/scripts/groupthink-script'
-        groupthink_dest = '{}/{}'.format(script_dir, org)
+        groupthink_dest = '{}/{}'.format(dest, alias)
         execute_cmd(['rm', groupthink_dest])
         execute_cmd(['cp', '-p', groupthink_script, groupthink_dest])
-        return 'Installed {} command.'.format(org)
+        return 'Installed {} command.'.format(group)
     except:
-        return "Could not get {0}-cli command. Please make sure https://github.com/{0}/{0}-cli exists.".format(alias)
+        return 'Could not get {0}-cli command. Please make sure https://github.com/{0}/{0}-cli exists.'.format(group)
```

Comment:
This should use alias. If a person sets an alternate name with `groupthink install blah --alias dcgov`, this would print `Please make sure https://github.com/blah/blah-cli exists`.